runtime/src/lib.rs

```diff
@@ -352,6 +350,10 @@ impl pallet_balances::Config for Runtime {
 	type WeightInfo = pallet_balances::weights::SubstrateWeight<Runtime>;
 	type MaxReserves = MaxReserves;
 	type ReserveIdentifier = [u8; 8];
+	type HoldIdentifier = ();
+	type FreezeIdentifier = ();
```

Comment:
why these two are nothing?
`HoldIdentifier` in [substrate](https://github.com/paritytech/substrate/blob/5e49f6e44820affccaf517fd22af564f4b495d40/bin/node/runtime/src/lib.rs#L465) 
`FreezeIdentifier` is () in substrate but should it be same here?

Reply:
We are not using these features in parachain. But I will update it.